Examples/Algorithms/TrackFitting/src/TrackFittingAlgorithm.cpp

```diff
@@ -74,13 +74,9 @@ ActsExamples::ProcessCode ActsExamples::TrackFittingAlgorithm::execute(
   // fit-function-object
   ActsExamples::MeasurementCalibrator calibrator(measurements);
 
-  GeneralFitterOptions options{ctx.geoContext,
-                               ctx.magFieldContext,
-                               ctx.calibContext,
-                               calibrator,
-                               &(*pSurface),
-                               Acts::LoggerWrapper{logger()},
-                               Acts::PropagatorPlainOptions()};
+  GeneralFitterOptions options{
+      ctx.geoContext, ctx.magFieldContext, ctx.calibContext,
+      calibrator,     &(*pSurface),        Acts::PropagatorPlainOptions()};
```

Comment:
I think this needs to be reformatted

Reply:
Seems like clang-format wants it this way, for some reason.